packages/blade/package.json

```diff
@@ -37,16 +37,25 @@
     "prepublish": "yarn build",
     "build": "run-s build:*",
     "build:clean": "rm -rf build",
-    "build:generate-types": "tsc -p ./tsconfig-generate-types.json",
-    "build:generate-css-variables": "GENERATE_CSS_VARIABLES=true FRAMEWORK=REACT rollup -c && node ./scripts/generateCssThemeVariables.js",
-    "build:copy-declarations": "copyfiles -u 1 \"src/**/*.d.ts\" build/types",
+    "// Generate types & move type declarations": "",
```

Comment:
:p poor-mans-packagejson-comments. 

Let me know if it's fine otherwise I'll remove it.